scripts/specializations/CpAIWorker.lua

```diff
@@ -484,16 +495,86 @@ function CpAIWorker:stopCpDriver()
 	if actionController ~= nil then
 		actionController:resetCurrentState()
 	end
-
 	self:raiseAIEvent("onAIFieldWorkerEnd", "onAIImplementEnd")
-
 end
 
 function CpAIWorker:getCpDriveStrategy()
     local spec = self.spec_cpAIWorker
     return spec.driveStrategy
 end
 
+function CpAIWorker:getCpReverseDrivingDirectionNode()
+    local spec = self.spec_cpAIWorker
+    if not spec.reverseDrivingDirectionNode and SpecializationUtil.hasSpecialization(ReverseDriving, self.specializations) then
+        spec.reverseDrivingDirectionNode =
+            CpUtil.createNewLinkedNode(self, "realReverseDrivingDirectionNode", self:getAIDirectionNode())
+        setRotation(spec.reverseDrivingDirectionNode, 0, math.pi, 0)
+    end
+    return spec.reverseDrivingDirectionNode
+end
+
+--- Fixes the ai reverse node rotation for articulated axis vehicles,
+--- if the node is pointing backwards and not forwards.
+function CpAIWorker:getAIReverserNode(superFunc)
```

Comment:
doesn't this make AIUtil.getArticulatedAxisVehicleReverserNode and its usage obsolete?

Reply:
Same as the Reverse questions above, should probably have another look at the reverse code in a separete PR